Cargo.toml

```diff
@@ -87,7 +87,7 @@ members = [
 ]
 
 [features]
-default = ["jitdump", "wasmtime/wat", "wasmtime/parallel-compilation", "wasi-nn"]
```

Comment:
@cfallin, I fixed the `openvino` crates so the error you saw above should be resolved. I think this line should be unnecessary if you bump the `openvino` crate to v0.3.2.

Reply:
Thanks!